Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports_description.md

```diff
@@ -0,0 +1,6 @@
+#### How to retrieve a Palo Alto Networks WildFire Reports API key
```

Comment:
add a note regarding having that in the license for TIM users. @ShirleyDenkberg FYI.

Reply:
Done.  Added the following 
*Note: If you have a TIM license that is up-to-date and are running Cortex XSOAR version 6.5, you do not need to set up an API key. The required information is automatically retrieved from your Cortex XSOAR TIM license.*